src/mf/mf-uvc.cpp

```diff
@@ -288,6 +298,13 @@ namespace librealsense
             if (hr == DEVICE_NOT_READY_ERROR)
                 return false;
 
+            if (hr == MF_E_SHUTDOWN_ERROR)
+            {
+                close_all();
```

Comment:
What is the expected flow n case of exception during call to `close_all`, will it be intercepted and handled in sensor?
Add to log